recipes/geotiff/bld.bat

```diff
@@ -0,0 +1,23 @@
+
+:: Had to set TIFF_NAMES, PROJ4_NAMES and JPEG_NAMES to force use of shared libs
+:: Note that zlib and jpeg only get linked in if tiff is linked statically, otherwise
+:: they don't seem to be used (presumeably they are used via libtiff in a dll build).
+cmake -G "NMake Makefiles" ^
```

Comment:
What I love about your PRs to staged-recipes is that you never skip Windows :wink: